### DISCUSSION TITLE
element.int not calling for all components from version Livewire 3.4.0 onwards

### DISCUSSION THREAD

**Discussion by sharat-rhinofish** [livewire/livewire#8022] [1. Help]
The following code
        document.addEventListener("DOMContentLoaded", () => {
            Livewire.hook('element.init', (component) => {
                // Note: element is a field inside component. Hence need to use component.el
                console.log(component.el.classList);
            });
        });

is not being executed for all components from version 3.4.0 onwards. Any ideas why?

**Comment by danie-ramdhani** [comment]
try using `livewire:init`:

```html
<script>
    // run after livewire inited
    document.addEventListener('livewire:init', () => {
        Livewire.hook('element.init', (component) => {
            // Note: element is a field inside component. Hence need to use component.el
            console.log(component.el.classList);
        });
    });
</script>
```

**Comment by sharat-rhinofish** [comment]
@danie-ramdhani , thanks for responding and apologies for the delay in responding. I tried what you suggested.

livewire:init as well as DOMContentLoaded works fine with Laravel Livewire upto version 3.3.5. However, with version 3.4 onwards, this does not work. I tried with the latest version of Livewire and the event is not being triggered.
